lib/fastlane/plugin/mango/helper/docker_commander.rb

```diff
@@ -35,6 +39,26 @@ def self.disconnect_network_bridge(container_name:)
       rescue StandardError
         # Do nothing if the network bridge is already gone
       end
+
+      def self.prune
+        Action.sh('docker system prune -f')
+      end
+
+      def self.handle_thin_pool_exception(&block)
+        begin
+          block.call
+        rescue FastlaneCore::Interface::FastlaneShellError => exception
+          if exception.message =~ /Create more free space in thin pool/
+            retry_counter = retry_counter.to_i + 1
+            if retry_counter < 2
+              prune
+              retry
```

Comment:
So, If we tried it 3 times it will just silently fail? Is this what we want or should we reraise the expection if we tried it 3 times?